ArduPlane/quadplane.cpp

```diff
@@ -637,7 +636,9 @@ bool QuadPlane::setup(void)
       that the objects don't affect the vehicle unless enabled and
       also saves memory when not in use
      */
-    motor_class = (enum AP_Motors::motor_frame_class)frame_class.get();
+    motor_class = (AP_Motors::motor_frame_class)frame_class.get();
+    motor_type  = (AP_Motors::motor_frame_type)frame_type.get();
+    bool coptar_tailsitter_supported = true;
```

Comment:
```suggestion
    bool copter_tailsitter_supported = true;
```

Reply:
wow, my eyes are getting worse :(
took forever to find the difference between those 2 strings